libcrux-hmac/src/hmac.rs

```diff
@@ -7,7 +7,13 @@ extern crate alloc;
 
 use alloc::vec::Vec;
 
-#[cfg(feature = "hacl")]
+#[cfg(all(feature = "hacl", not(feature = "expose-hacl")))]
+mod hacl {
+    pub(crate) mod hash_sha1;
+    pub(crate) mod hmac;
+}
+
+#[cfg(feature = "expose-hacl")]
```

Comment:
Ah, that's the feature :)
Let's throw the names for features etc. somewhere on some doc to have some guidelines how we want these things to work.